meta-balena-common/recipes-support/hostapp-update-hooks/files/0-signed-update

```diff
@@ -123,13 +123,16 @@ updateKeys() {
     RESULT_DIR="$(mktemp -d)"
     CURRENT_POLICY_PATH="$(find /mnt/efi -name "policies.*")"
     for UNLOCK_PCRS in  0,2,3,7 0,1,2,3; do
-        [ -f "${SESSION_CTX}" ] && tpm2_flushcontext "${SESSION_CTX}" || true
-        tpm2_startauthsession --policy-session -S "${SESSION_CTX}"
-	    tpm2_policypcr -S "${SESSION_CTX}" -l "sha256:${UNLOCK_PCRS}"
-	    POLICIES="$(find "${CURRENT_POLICY_PATH}" -type f | sort | xargs)"
-	    if [ "$(echo "${POLICIES}" | wc -w)" -gt 1 ]; then
-            tpm2_policyor -S "${SESSION_CTX}" "sha256:$(echo "${POLICIES}" | sed 's/ /,/g')"
-	    fi
+        {
+            [ -f "${SESSION_CTX}" ] && tpm2_flushcontext "${SESSION_CTX}"  || true
+            tpm2_startauthsession --policy-session -S "${SESSION_CTX}"
+            tpm2_policypcr -S "${SESSION_CTX}" -l "sha256:${UNLOCK_PCRS}"
+            POLICIES="$(find "${CURRENT_POLICY_PATH}" -type f | sort | xargs)"
+            if [ "$(echo "${POLICIES}" | wc -w)" -gt 1 ]; then
+                tpm2_policyor -S "${SESSION_CTX}" "sha256:$(echo "${POLICIES}" | sed 's/ /,/g')"
+            fi
+
+        } >/dev/null 2>&1
```

Comment:
Do we want to silence both stdout and stderr? Wouldn't stderr be silent by default unless there are actual errors, and then we will want to see those?

Reply:
Good question, @alexgg. I think it's really only helpful for silencing `tpm2_flushcontext`, which we expect to fail when there's no active auth context. I think we can remove the redirection for stderr from the command grouping, and only add it to this one command. I'll check this.